addons/im_livechat/static/src/core/web/transcript_sender_patch.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="im_livechat.TranscriptSender" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-livechat-TranscriptSender')]" position="replace">
+            <ActionPanel title.translate="Send a copy of conversation" resizable="false" icon="'fa fa-paper-plane'">
+                <t>$0</t>
+            </ActionPanel>
+        </xpath>
+    </t>
```

Comment:
It's a bit odd to replace the transcript sender in the backend: it would prevent us from using this component inside the chat window's feedback panel, as we do in the embed context. We'll most likely want to support continuing live chats from the backend in the future. I'd prefer having a separate component instead.